packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -17,6 +18,8 @@ import { assert, performance } from "@fluidframework/common-utils";
  */
 export const latencyThreshold = 5000;
 
+export const numOpsToThrottle = 100;
```

Comment:
The name of this constant seems wrong - I do not think we throttle anything.
Also 100 is too low, I'd use 500 or 1000 - this gives us plenty of data.

Reply:
I ended up forgetting to revert it - I was using 100 for testing. Also, removed the const.